### ISSUE TITLE
How can self.log_std be updated?

### ISSUE THREAD

**Issue by zhengsizuo** [higgsfield/RL-Adventure-2#23]
self.log_std= nn.Parameter(torch.ones(1, num_outputs) * std)
I konw the final policy loss is related to self.logstd, but when I print the log_std every step, it showed 0.0001 increase per step, why?
